router/types.go

```diff
@@ -11,24 +11,6 @@ import (
 	"github.com/rudderlabs/rudder-server/utils/misc"
 )
 
-// JobParameters struct holds source id and destination id of a job
-type JobParameters struct {
-	SourceID                string      `json:"source_id"`
-	DestinationID           string      `json:"destination_id"`
-	ReceivedAt              string      `json:"received_at"`
-	TransformAt             string      `json:"transform_at"`
-	SourceTaskRunID         string      `json:"source_task_run_id"`
-	SourceJobID             string      `json:"source_job_id"`
-	SourceJobRunID          string      `json:"source_job_run_id"`
-	SourceDefinitionID      string      `json:"source_definition_id"`
-	DestinationDefinitionID string      `json:"destination_definition_id"`
-	SourceCategory          string      `json:"source_category"`
-	RecordID                interface{} `json:"record_id"`
-	MessageID               string      `json:"message_id"`
-	WorkspaceID             string      `json:"workspaceId"`
-	RudderAccountID         string      `json:"rudderAccountId"`
```

Comment:
as for `RudderAccountId`, it is present in multiple structs in the repo.
I will tackle them in another PR. Or perhaps defer to integrations, since they seem to have introduced it.

Reply:
RudderAccountID is used for OAuth destination IIRC

Reply:
Nobody is reading it in server's code though, @sanpj2292 ?